cannon/mipsevm/testutil/mips.go

```diff
@@ -80,6 +80,7 @@ func (m *MIPSEVM) Step(t *testing.T, stepWitness *mipsevm.StepWitness, step uint
 	stateHash, err := stateHashFn(evmPost)
 	require.NoError(t, err, "state hash could not be computed")
 	require.Equal(t, stateHash, postHash, "logged state must be accurate")
+	//t.Logf("Input: %x\nExpectedHash: %x\n", input, postHash)
```

Comment:
DEBUGME

Reply:
This is kinda useful - was using it to update the solidity test `test_mips2_step_debug_succeeds` when the state format changed.  Though I guess that doesn't happen very often ... 🤔